core/chains/evm/config/toml/defaults/Hedera_Mainnet.toml

```diff
@@ -0,0 +1,33 @@
+ChainID = '295'
+ChainType = 'hedera'
+#considering the 3-5 (6 including a buffer) seconds of finality and 2 seconds block production
+#we set the depth to 6/2 = 3 blocks, setting to 10 for safety
+FinalityDepth = 10
+#Hedera has high TPS, so polling less often 
+LogPollInterval = '10s'
+MinIncomingConfirmations = 1
+#turning off the liveness detection, since the Websocket connection is not reliable, can be turned on if needed
+NoNewHeadsThreshold = '0'
+
+[BalanceMonitor]
+Enabled = true
+
+[GasEstimator]
+Mode = 'SuggestedPrice'
+# since Hedera dont have mempool and there's no way for a node to front run or a user to bribe a node to submit the transaction earlier than it's consensus timestamp
+# but they have automated congesting pricing throttling which would mean at high sustained level the gasPrice itself could be increased to prevent malicious behaviour.
+# so wait for 3 blocks before bumping the gas price, the increased gas Price could be adjusted
+BumpThreshold = 3
```

Comment:
Our current logic checks the on-chain nonce immediately after broadcasting. If it hasn't incremented, we immediately rebroadcast with a bumped transaction without waiting `BumpThreshold` amount of blocks since it's on initial broadcast. I feel like this could be set to 0. If we ever reach our confirmation logic, the transaction wouldn't need bumping since it should be included on-chain because of instant finality unless I'm mistaken.

Reply:
Ah I see, this is only for Hedera? with the assumption that it doesnt have a mempool, Yea I think it could be disabled

Reply:
Ya that logic is specific to Hedera to avoid the nonce gap issue we had encountered.